spyder/config/appearance.py

```diff
@@ -0,0 +1,641 @@
+import os
+import sys
+
+from spyder.config.fonts import MEDIUM, MONOSPACE, SANS_SERIF, SMALL
+
+WIN = os.name == 'nt'
+LINUX = sys.platform.startswith('linux')
+
+# Default themes
+APPEARANCE = {
+              'icon_theme': 'spyder 3',
```

Comment:
Please only leave four spaces of indentation between the left margin of the file and this line, and align all the rest of the lines on this dict accordingly.